ocp_install_env.sh

```diff
@@ -95,20 +88,22 @@ metadata:
   name: ${CLUSTER_NAME}
 compute:
 - name: worker
-  replicas: 0
+  replicas: $NUM_WORKERS
 controlPlane:
   name: master
   replicas: ${NUM_MASTERS}
   platform:
     baremetal: {}
 platform:
   baremetal:
+    provisioningNetworkInterface: "enp1s0"
```

Comment:
@hardys The default in openshift/installer is `ens3`, but that's wrong for dev-scripts. I wonder if we should even have a default value for this field, as it's really hardware and environment-specific, what do you think?

Reply:
Yeah I guess it makes sense to remove the default, I commented on the installer PR though as I think we have to make it very clear what the needed info is there when folks invariably don't initially include it.